source/glest_game/global/core_data.cpp

```diff
@@ -61,15 +61,16 @@ namespace Glest {
 
 		static const string CORE_WATER_SOUNDS_PATH = CORE_PATH + "/water_sounds/";
 
+		static const string data_path = getGameReadWritePath(GameConstants::path_data_CacheLookupKey);
```

Comment:
This almost compiles but I get a weird failure message

```
[ 67%] Linking CXX executable ../../../zetaglest
[100%] Built target zetaglest
help2man: can't get `--help' info from /home/andy/src/ZetaGlest/zetaglest-source/mk/linux/zetaglest
Try `--no-discard-stderr' if option outputs to stderr
source/glest_game/CMakeFiles/zetaglest.6.dir/build.make:57: recipe for target 'source/glest_game/CMakeFiles/zetaglest.6' failed
make[2]: *** [source/glest_game/CMakeFiles/zetaglest.6] Error 139
CMakeFiles/Makefile2:232: recipe for target 'source/glest_game/CMakeFiles/zetaglest.6.dir/all' failed
make[1]: *** [source/glest_game/CMakeFiles/zetaglest.6.dir/all] Error 2
Makefile:105: recipe for target 'all' failed
make: *** [all] Error 2
ERROR: MAKE failed.
```

When I replaced the line with

`static const string data_path = "/home/andy"`

The build completes and I run ZG but of course I get errors when it looks for the data (just tried that for debugging).